build/zip_bundle.gni

```diff
@@ -19,11 +19,25 @@ if (flutter_runtime_mode == "jit_release") {
 # The files variable is an array of scopes that specify a source file or
 # directory and a destination path in the archive to create.
 #
+# If this zip file is not intended to be codesigned as a Mac artifact,
```

Comment:
This is probably unnecessary if we can rely on the entitlements lists to decide whether to generate files or not.